src/components/CameraControls/index.tsx

```diff
@@ -208,13 +224,14 @@ const CameraControls = ({
                     active={cameraProjectionType === PERSPECTIVE}
                 />
             </div>
-            <ViewportButton
-                tooltipText={"Home view (H)"}
-                tooltipPlacement="left"
-                icon={getIconGlyphClasses(IconGlyphs.Reset)}
-                clickHandler={resetCamera}
-            />
+            {homeViewButton}
         </div>
     );
+
+    const renderControls = minimalControls
```

Comment:
nit but `minimalControls` and `mininalEmbeddedControls` are really close in name. I would probably change the prop to `showFullControls` for clarity.